scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -4,21 +4,33 @@
 
 # Pad targets in the target file by the specified amount (this was found to improve sensitivity and specificity)
 task PadTargets {
-    File? targets
+    File targets
     Int? padding
-    File gatk_jar
+    String gatk_jar
+
+    # Runtime parameters
     Int? mem
+    String gatk_docker
+    Int? preemptible_attempts
+    Int? disk_space_gb
 
     # Determine output filename
     String filename = select_first([targets, ""])
-    String base_filename = sub(sub(sub(filename, "gs://", ""), "[/]*.*/", ""), "\\.tsv$", "")
+    String base_filename = basename(filename, ".tsv")
 
-    command {
+    command <<<
         echo ${filename}; \
-        java -Xmx${default=1 mem}g -jar ${gatk_jar} PadTargets \
+        java -Xmx${default="1" mem}g -jar ${gatk_jar} PadTargets \
             --targets ${targets} \
-            --padding ${default=250 padding} \
+            --padding ${default="250" padding} \
             --output ${base_filename}.padded.tsv
+    >>>
+
+    runtime {
+        docker: "${gatk_docker}"
+        memory: select_first([mem, 2]) + " GB"
```

Comment:
No problem to leave it for now, but it seems like `default` should be canonical.